docs/custom-webview-android.md

```diff
@@ -177,6 +296,30 @@ public class CustomWebViewManager extends RNCWebViewManager {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+class CustomWebViewManager : RNCWebViewManager() {
+    override fun getExportedCustomDirectEventTypeConstants(): MutableMap<Any?, Any?>? {
+        val export =
+            if (super.getExportedCustomDirectEventTypeConstants() != null) {
+                super.getExportedCustomDirectEventTypeConstants()
+            } else {
+                MapBuilder.newHashMap<Any, Any?>()
+            }
+        export?.set(
+            "navigationCompleted",
+            MapBuilder.of("registrationName", "onNavigationCompleted")
+        )
```

Comment:
```suggestion
        export["navigationCompleted"] = 
            MapBuilder.of("registrationName", "onNavigationCompleted")
```

Reply:
It shows error "only safe or non-null asserted calls are allowed on receiver"

Reply:
It should work once you refactor the rest of the code as `export` will be non nullable

Reply:
Nice, done